### PR TITLE
feat: Add dry streak message

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

**Comment by kristofbolyai** [comment]
image

**Comment by magicus** [comment]
> image

Oh, you're a lucky one! ;-)

File: common/src/main/java/com/wynntils/features/debug/PacketDebuggerFeature.java [hunk 1]
```diff
@@ -83,6 +86,7 @@ private String describePacket(Packet<?> packet) {
     public void onPacketSent(PacketSentEvent<?> e) {
         Packet<?> packet = e.getPacket();
         if (IGNORE_LIST.contains(packet.getClass())) return;
+        if (!(packet instanceof ServerboundContainerClickPacket)) return;
```
Comment (magicus) [inline common/src/main/java/com/wynntils/features/debug/PacketDebuggerFeature.java]: debug stuff left over..?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/debug/PacketDebuggerFeature.java]: Ah, from container query stuff...

**Comment by magicus** [comment]
So the chest count never resets? Is this obvious what the "chest #8913" mean? Is this how it is presented in Legacy?

**Comment by kristofbolyai** [comment]
> So the chest count never resets? Is this obvious what the "chest #8913" mean? Is this how it is presented in Legacy?

This is "extended" legacy parity. Legacy only displayed dry count and that's it. 

I based this off [this image](https://cdn.discordapp.com/attachments/661316274735022120/1093314150446411896/image.png) (1.12.2 [LootrunStatistics](https://github.com/McPlayHD/LootrunStatistics) mod). I am planning to bring Artemis to feature parity. 

(A while back I had talks with the mod's dev, McPlayHD, and they were happy to merge their mod into Artemis, at the time, even trying to port stuff themselves. I haven't talked to them since, but most likely university took their time.)

File: common/src/main/java/com/wynntils/models/containers/LootChestModel.java [hunk 2]
```diff
@@ -37,16 +38,21 @@ public int getDryCount() {
         return dryCount.get();
     }
 
-    public Integer getDryBoxes() {
+    public int getDryBoxes() {
         return dryBoxes.get();
     }
 
+    public int getChestCount() {
```
Comment (magicus) [inline common/src/main/java/com/wynntils/models/containers/LootChestModel.java]: Maybe name it getOpenedChestCount() instead?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/models/containers/LootChestModel.java]: Done

**Review by magicus — APPROVED** [review]

**Pull request merged**